docs/getting-started.adoc

```diff
@@ -3,9 +3,94 @@
 Following guide provides general information for starting with AeroGear Services SDK.
 If you looking for 
 
+== Prerequisites
+You will need the following installed on your system
+ * Android SDK or Android Studio from [Google](https://developer.android.com/studio/index.html)
+It is reccomended you have the following installed  
+* The latest version of [OpenJDK](http://openjdk.java.net/install/index.html)
+
+== Installing
+As of the writing of this guide, the Aerogear Android SDK is not in jcenter and needs to be installed manually.
+
+=== Clone and build the library
+```sh
+git clone https://github.com/aerogear/aerogear-android-sdk
+cd aerogear-android-sdk
+./gradlew install
+```
+
+You can verify the install went well by checking your local maven repository
+
+```
+ls ~/.m2/repository/org/aerogear
+
+#Should output auth  core  metrics
+
+```
+
+
+
+== Using the SDK
+
+=== MavenLocal()
+
+If you are using a build of the SDK from source you will need to add mavenLocal() to the build.gradle file in your project's root directory.
+
+```groovy
+allprojects {
+    repositories {
+        google()
+        jcenter()
+        mavenLocal() // <-- Add This line
```

Comment:
to help with testing the lib after we publish a new version, should this be added as the first repo?

Reply:
I'm not sure why it would help with testing, I think I might be missing something.  Can you elaborate?